### ISSUE TITLE
SEND L3 ERROR on mac os

### ISSUE THREAD

**Issue by RMS21** [Ettercap/ettercap#947]
I try to build and install from lates source code but i have the issue and not solved

<img width="817" alt="Screen Shot 2019-06-08 at 9 50 16 AM" src="https://user-images.githubusercontent.com/18057471/59142402-05356180-89d3-11e9-80d6-81d2e9417a03.png">

**Comment by koeppea** [comment]
Have you built in debug mode?

   cmake -DCMAKE_BUILD_TYPE=Debug -DENABLE_IPV6=On ..
   make
   sudo make install

If this is the case I’d be interested in the etterca_debug logfile.

Can you also provide a packet capture by starting ettercap with the -w parameter set?

**Comment by RMS21** [comment]
i follow the instruction in wiki so yes i run it in debug mode
here are the debug file and packets
[ettercap-0.8.2_debug.log](https://github.com/Ettercap/ettercap/files/3269311/ettercap-0.8.2_debug.log)

I couldn't upload the packets file with .pcap extenstion so i rename it to txt

[log.txt](https://github.com/Ettercap/ettercap/files/3269312/log.txt)

**Comment by koeppea** [comment]
Just checked.
Apparently this comes due to the IPv6 Router Advertisements your Home Router sends out periodically.
So this is nothing that would impact the functionality AFAICS. It's just a cosmetic issue.
I gonna check if I can reproduce this in my network....

**Comment by koeppea** [comment]
The Packets to `ff02::fb` only showed in the screenshot are mDNS packets, flooded in consumer level equipment. But again this should not be a defect, but just ugly.
Is there anything besides these messages that is not working?

**Comment by RMS21** [comment]
No, it was just the l3 errors and as i tested on another network this messages didn't show.
Thanks for your help
